bearer/bearer.go

```diff
@@ -364,10 +384,31 @@ func (b Token) SigningKeyBytes() []byte {
 	return nil
 }
 
+// SetIssuer sets NeoFS user ID of the Token issuer.
+//
+// See also [Token.Issuer], [Token.Sign].
+func (b *Token) SetIssuer(usr user.ID) {
+	b.issuerSet = true
+	b.issuer = usr
+}
+
+// Issuer returns NeoFS user ID of the Token issuer. Zero value means unset
+// issuer.
+//
+// See also [Token.SetIssuer], [Token.Sign].
+func (b Token) Issuer() user.ID {
+	if b.issuerSet {
+		return b.issuer
+	}
+	return user.ID{}
```

Comment:
Practically this means lots of
```
var iss = t.Issuer()
if iss == user.ID{} {
    iss = t.ResolveIssuer() // nolint:something
}
```
code. Maybe we can do resolving internally here?

Reply:
it's a field getter, let it be clear. There is already a `ResolveIssuer` for resolving

Reply:
Yeah, but it's deprecated and you're recommending to use `Token.Issuer`. And it won't work in some cases for old tokens. As an SDK user I want to get an issuer and I don't care much about the exact mechanism behind, if there is no way for me to do it I'll write these lines of code from above again and again (and use deprecated API). To me options are:
 * make it a part of `Issuer()`
 * make it a part of `ResolveIssuer()`
 * provide an additional method

I'd probably go with `Issuer()` handling this.

Reply:
it's not only a deprecated API but deprecated approach. I try to stop resolving issuer from key. If u need it anyway, there is https://pkg.go.dev/github.com/nspcc-dev/neofs-sdk-go@v1.0.0-rc.11/bearer#Token.SigningKeyBytes or `ResolveIssuer` (deprecated does not mean u cant use it, its not a vulnerability). No problem to me at all

Reply:
But you can have already issued tokens, so you will have to use `ResolveIssuer` anyway as a fallback.

Reply:
resolve or not resolve - app-side logic to me. I can agree that we aint ready to deprecate `ResolveIssuer`, but disagree with autofallback in `Issuer`. Having both, an app can implement any logic it needs

Reply:
now `ResolveIssuer` prechecks field presence